cartoframes/context.py

```diff
@@ -39,6 +39,13 @@
     mpi = None
     plt = None
 HAS_MATPLOTLIB = plt is not None
+try:
+    import geopandas
+    from shapely.geometry import Point
```

Comment:
Since we don't need `geopandas` or `shapely.geometry.Point` anywhere in `cartoframes`, how about testing whether they can be imported other than importing them.

Looks like there's a way to do that here: https://stackoverflow.com/questions/14050281/how-to-check-if-a-python-module-exists-without-importing-it

We'd probably need an:
```python
if version.major_version == 2:
    python2.7 version
elfi version.version < (3, 3):
    python3.3ish version
elif version.version >= (3, 4):
   python3.4+ stuff
else:
   HAS_GEOPANDAS = False
```